### PR TITLE
Set designer tools left spacing conditionally based on panel location

### PR DESCRIPTION
**Pull request by jaimereyesmx**
Allow extra spacing for when panel location is to the left

### CODE REVIEW

File: libs/designer/src/lib/ui/styles.less [hunk 1]
```diff
@@ -69,6 +69,10 @@
   box-sizing: border-box;
 }
 
+.msla-designer-tools-left-panel {
+  left: 52px !important; // Add extra 32px taken from the collapsed panel
```
Comment (hartra344) [inline libs/designer/src/lib/ui/styles.less:73]: Is it possible to do this without !important? I really like to make sure we go through every possible alternative before settling on !important

Comment (ek68794998) [reply libs/designer/src/lib/ui/styles.less:73]: If it doesn't take by default, changing the selector to `.msla-designer-tools.msla-designer-tools-left-panel` should cause it to overwrite the base `.msla-designer-tools` styles.

Comment (jaimereyesmx) [reply libs/designer/src/lib/ui/styles.less:73]: Will try that

Comment (jaimereyesmx) [reply libs/designer/src/lib/ui/styles.less]: Fixed it

**Review by ek68794998 — APPROVED** [review]

**Pull request merged**